motor_monitor_and_vibration/README.md

```diff
@@ -0,0 +1,184 @@
+
+Set the ProductUID for the Notecard by pasting the command below into the in-browser terminal. Make sure to replace `com.your-company:your-product-name` with the ProductUID from your Notehub project, which can be found below your project's name in the dashboard at https://notehub.io. Also, replace the placeholder serial number (`"sn":"motor-location"`) with the location or other identifier for your motor.
+
+```json
+{ "req": "hub.set", "mode": "periodic", "outbound": 5, "product": "com.your-company:your-product-name", "sn": "motor-location" }
+```
+
+This app uses `periodic` mode to minimize power use so that the solution can remain online for longer when mains power fails. `"outbound": 5` is the max wait time, in minutes, to sync outbound data from the Notecard. For more details, see [our Essential Requests documentation for `hub.set`](https://dev.blues.io/notecard/notecard-walkthrough/essential-requests/#notehub-configuration).
+
+### Configure Vibration Sensing
+
+Vibration sensing is configured using a number of distinct requests. Each request given below should be pasted into the in-browser terminal separately.
+
+#### `card.motion.mode`
+
+The [`card.motion.mode`](https://dev.blues.io/reference/notecard-api/card-requests/#card-motion-mode) request enables motion sensing using the Notecard's built-in accelerometer. Motion is used to determine how much the motor is vibrating. The configuration below uses maximum sensitivity (`5`) and a 5 second period for each bucket of movement count.
+
+```json
+{
+  "req":          "card.motion.mode",
+  "start":        true,
+  "sensitivity":  5,
+  "seconds":      5
+}
+```
+
+#### `card.motion.track`
+
+The [`card.motion.track`](https://dev.blues.io/reference/notecard-api/card-requests/#card-motion-track) request
+configures the Notecard to write motion notes to notefile `_motion.qo` every 5 minutes. Each motion bucket (configured in `card.motion.mode` above) is 5 seconds of motion, so we use 60 of them to fill the 5 minute interval.
+
+```json
+{
+  "req":       "card.motion.track",
+  "minutes":   5,
+  "count":     60,
+  "start":     true
+}
+```
+
+### `card.motion.sync`
+
+The [`card.motion.sync`](https://dev.blues.io/reference/notecard-api/card-requests/#card-motion-sync
+) request configures how often the Notecard syncs tracking data to notehub. Here we use 5 minutes. Adjust this
+according to how much data you wish to use and the responsiveness of alerts required.
+
+```json
+{
+  "req":      "card.motion.sync",
+  "start":    true,
+  "minutes":  5
+}
+```
+
+### `card.aux - gpio mode`
+
+The [`card.aux`](https://dev.blues.io/notecard/notecard-walkthrough/advanced-notecard-configuration/#monitoring-aux-gpio-state-with-environment-variables) request configures how the Notecard uses the auxillary GPIO pins.  Here we wish to either monitor or control the motor - which you use depends upon whether the motor provides the activity signal or whether the Notecard controls the motor's activity.
```

Comment:
Looks like auxillary is still misspelled.

Reply:
l -> i done :-)